client/lib/text-utils/index.js

```diff
@@ -24,4 +30,26 @@ function countWords( content ) {
 	return 0;
 }
 
-export default { countWords };
+const diffWords = JsDiff.diffWords;
```

Comment:
removed it completely as we're directly importing (and re-exporting in the "default" object `diffWords`)